config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  namespace :api, defaults: { format: :json } do
+    namespace :v1 do
+      get 'random', to: 'greetings#random'
```

Comment:
- [ ] Kindly endeavor to install and configure CORS gems so as to resolve this error, as recommended in the previous code review.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/160798002-b6108163-2f74-4030-af5a-b0acdb24347b.png)